development/node-helper.md

```diff
@@ -147,12 +147,16 @@ module.
 
 ### `this.sendSocketNotification(notification, payload)`
 
-**_notification_ String** - The notification identifier.<br> **_payload_
-AnyType** - Optional. A notification payload.<br>
+**_notification_ String** - The notification identifier.<br>
```

Comment:
Do we need those <br>s at all? Shouldnt an extra blank line do the same trick without html tags in the markdown?

Reply:
now, we have develop doc website, we can do some test :)

Reply:
I push all to develop for see, and will discover react

Reply:
Note: I see , this result in other file

If you make a blank line you will have a blank line and not a `<br>`

Reply:
Don't forget that you can also run vuepress locally which allows you to see a realtime preview.

```bash
npm run docs:dev
```

Reply:
![image](https://github.com/MichMich/MagicMirror-Documentation/assets/30669209/c9fe0c99-c3fe-438c-974d-ba102e6f2015)

I should install it, maybe it will help

Reply:
> I should install it, maybe it will help

like i says :)

Reply:
without this `<br>`
![image](https://github.com/MichMich/MagicMirror-Documentation/assets/30669209/f94af351-8d8b-4b90-b1ac-1df8b6df28d7)

with it:

![image](https://github.com/MichMich/MagicMirror-Documentation/assets/30669209/9d50cf40-6e7c-436b-96fd-2419e5da8d1c)


we have the response :)